PlumAnalyticsSettingsForm.inc.php

```diff
@@ -89,7 +89,7 @@ function readInputData() {
 	 * Fetch the form.
 	 * @copydoc Form::fetch()
 	 */
-	function fetch($request) {
+	function fetch($request, $template = NULL, $display = false) {
```

Comment:
Be sure to pass these arguments along to the `parent::fetch()` call.

Reply:
Good catch. I've addressed in [2235d55](https://github.com/ulsdevteam/ojs-plum-plugin/pull/13/commits/2235d55410c9fc5fabb078dc9689707dcbac8c64)